news/serialization-harmonization.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* ``GufeTokenizable.from_json`` now falls back to loading ``dict`` representation if from ``keyed_chain`` fails
+
+**Deprecated:**
+
+* ``Transformation.dump``, ``Transformation.load`` are now deprecated
```

Comment:
```suggestion
* ``Transformation.dump``, ``Transformation.load`` are now deprecated, use Transformation.to_json and Transformation.from_json instead
```

Are these drop-in replacements?

Reply:
Effectively, yes. They use the `keyed_chain` representation, but `from_json` will fall back to assuming the `dict` representation `dump` produced if assuming a `keyed_chain` fails.